### PR TITLE
Fix incorrect CORS default values

### PR DESCRIPTION
**Pull request by wxiaoguang**
Document: 

```
;ALLOW_DOMAIN = *
;METHODS = GET,HEAD,POST,PUT,PATCH,DELETE,OPTIONS
```

### CODE REVIEW

File: modules/setting/cors.go [hunk 1]
```diff
@@ -21,9 +21,10 @@ var CORSConfig = struct {
 	Headers          []string
 	XFrameOptions    string
 }{
-	Enabled:       false,
```
Comment (silverwind) [inline modules/setting/cors.go:24]: Uninitualized bool defaults to false, right? Maybe keep it for explicity reasons.

Comment (wxiaoguang) [reply modules/setting/cors.go:24]: No reason, there is also a `AllowSubdomain`, no need to fill every empty fields.

**Review by silverwind — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by silverwind** [comment]
.

**Comment by delvh** [comment]
Should we mark this PR as breaking so that we will not forget it when writing the release blog?

**Comment by wxiaoguang** [comment]
At least, I don't see it would affect end users.

**Comment by silverwind** [comment]
Previous default config did not set any CORS headers, right? I recall CORS config is confusingly split into at least 2-3 config options, some work on some endpoints, some on another.

**Comment by wxiaoguang** [comment]
> Previous default config did not set any CORS headers, right?

I guess yes. before if there is only `ENABLED=true`, then CORS seems not working.

But I think this PR only fixes the bug, it doesn't really introduce "breaking" behavior because the document already said that `ALLOW_DOMAIN="*"`

**Pull request merged**